<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inicio de sesión</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QPushButton" name="ok_button">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="employee_forgot_password_label">
     <property name="text">
      <string>¿Olvidó su contraseña?&lt;br&gt;Comuníquese con un gerente para cambiar su contraseña.</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="manager_forgot_password_label">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>¿Olvidó su contraseña?&lt;br&gt;Comuníquese con Soporte Técnico para cambiar su contraseña.</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>RUT</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="rut_input"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Contraseña</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="password_input">
         <property name="echoMode">
          <enum>QLineEdit::EchoMode::Password</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="show_password_button">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>assets/basicons/eye-password.svg</normaloff>assets/basicons/eye-password.svg</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
